tests/acceptance/brocfile-smoke-test-slow.js

```diff
@@ -1,427 +0,0 @@
-'use strict';
```

Comment:
This file should not be removed. Even though it is named “brocfile” it is about ember-cli-build.js mostly (you can see that if you review the deleted fixtures)...

Reply:
@rwjblue fixed